.github/workflows/deploy-docs.yml

```diff
@@ -17,28 +17,30 @@ permissions:
 jobs:
   run:
     runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash -l {0}
 
     steps:
     - uses: actions/checkout@v4
       with:
         fetch-depth: 0
-        ref: main
+        # ref: main
```

Comment:
@Conengmo this is the culprit. I'm not sure if we need this though.

xref,: https://github.com/python-visualization/folium/pull/1965/files#r1808652059

Reply:
good catch! Pretty sure we don't need that. We already have if-statements in that workflow to make sure we do certain things only on main branch or on release. I think it got in there because I missed it in review, my bad!

```suggestion
```